vpr/src/route/route_timing.cpp

```diff
@@ -1218,7 +1219,7 @@ static void timing_driven_expand_neighbours(t_heap *current,
         t_bb bounding_box, float bend_cost, float criticality_fac,
         int num_sinks, int target_node,
         float astar_fac, int highfanout_rlim, route_budgets& budgeting_inf, float max_delay, float min_delay,
-        float target_delay, float short_path_crit, RouterStats& router_stats) {
+        float target_delay, float short_path_crit, RouterStats& router_stats, bool is_global_net) {
```

Comment:
To simplify timing_driven_expand_neighbours() and timing_driven_route_connection():

* Convert timing_driven_expand_neighbours to always use the passed in bounding_box
  * Eliminates num_sinks, highfanout_rlim and is_global_net arguments
* Similarly in timing_driven_route_connection()

* Instead timing_driven_route_sink() will figure out what the correct bounding box to use (either net bounding box, or high fanout target-based bounding box, depending on net fanout).

Reply:
Done: You pushed a commit that already fixed this and I successfully merged it